.github/workflows/typos.toml

```diff
@@ -1,6 +1,5 @@
 [default.extend-identifiers]
 alog = "alog"
-Jetbrains = "JetBrains"
```

Comment:
Is this correct?

Reply:
I had to remove this to fix the lint check otherwise, typos cli complains about
https://github.com/coder/coder/pull/10491/files#diff-e2fb60d4b313fc2ad42c37ce065c0c880e04615b339a843e14892338d0bb4396R268

There ıs no way to skıp checkıng a single line. 
The best we can do is skip the whole file.

Reply:
Ah, I see. And this is a github linter, no golang-ci lint. That you could skip one line.